lms/templates/main.html

```diff
@@ -107,13 +107,13 @@
 % if not disable_window_wrap:
   <div class="window-wrap" dir="${static.dir_rtl()}">
 % endif
-    <a class="nav-skip" href="<%block name="nav_skip">#content</%block>">${_("Skip to main content")}</a>
+    <a class="nav-skip" href="#content">${_("Skip to main content")}</a>
```

Comment:
@cahrens I'm removing the dynamic construction of this, which guarantees it syncs up with the skip link. Is this okay?